azure_storage/config.yaml

```diff
@@ -8,12 +8,23 @@ options:
   storage-account:
     type: string
     description: The name of the Azure storage account.
+  resource-group:
+    type: string
+    description: |
+      The name of the Azure resource group where the storage account is located. 
+      This is optional and can be used to specify a different resource group.
+    default: ''
+  endpoint:
+    type: string
+    description: |
+      The endpoint URL for the Azure Storage account. This is optional and can be 
+      used to override the default endpoint URL.
```

Comment:
suggestion(blocking): Let's give a little bit more context on how this value will be used.

I was thinking potentially something like:
```
The endpoint URL for the Azure Storage account. This is optional and can be 
used to override the default endpoint URL, that would be generated based
on the provided connection-protocol, container and storage-account.
``` 
```